apps/nully/app/[locale]/page.tsx

```
@@ -1,9 +1,9 @@
 import { getTranslations } from "next-intl/server";
 import { type Locale } from "next-intl";
-import { Check, Database, Infinity, Layers, Zap, Upload, Globe } from "lucide-react";
+import { Check, Database, Infinity, Layers, Zap, Upload, Shield, CheckCircle, MapPin } from "lucide-react";
```

Biome (2.1.2)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)